### PR TITLE
#3340 - Make any atom and atom list part of the Atom properties

### PR DESCRIPTION
**Pull request by AKZhuk**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [ ] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

**Review by MartaWilliams — APPROVED** [review]

**Review by OlgaMazurina — APPROVED** [review]
Good job!

File: packages/ketcher-core/src/domain/constants/generics.ts [hunk 1]
```diff
@@ -267,3 +267,18 @@ export const Generics: GenericsType = {
     },
   },
 };
+
+function getGenericsList(generics) {
```
[the remark this answers is not in the document]
Comment (AKZhuk) [reply packages/ketcher-core/src/domain/constants/generics.ts]: done

File: packages/ketcher-core/src/application/render/restruct/reatom.ts [hunk 2]
```diff
@@ -1103,6 +1101,28 @@ function getSubstitutionCountAttrText(value: number) {
   return attrText;
 }
 
+function getAtomLabelAttrText(value: string, atom) {
```
[the remark this answers is not in the document]
Comment (AKZhuk) [reply packages/ketcher-core/src/application/render/restruct/reatom.ts:1104]: After discussion with the author, we decided to leave it as is.

**Comment by ViktoriaTkacheva** [comment]
Preview. No comments

**Pull request merged**